src/main/scala/com/atomist/rug/function/github/CreateTagFunction.scala

```diff
@@ -35,15 +31,14 @@ class CreateTagFunction
     logger.warn(s"Invoking createTag with tag '$tag', message '$message', sha '$sha', owner '$owner', repo '$repo', apiUrl '$apiUrl' and token '${safeToken(token)}'")
 
     val apiUrlSlash = if (apiUrl.endsWith("/")) apiUrl else s"$apiUrl/"
+    val ghs = gitHubServices(token, apiUrlSlash)
```

Comment:
I would have expected this to be `GitHubServices(token, apiUrlSlash)`, and I see it that way other places. What's going on?